openpype/modules/deadline/deadline_module.py

```diff
@@ -32,3 +44,37 @@ def get_plugin_paths(self):
         return {
             "publish": [os.path.join(current_dir, "plugins", "publish")]
         }
+
+    @staticmethod
+    def get_deadline_pools(webservice, log=None):
+        # type: (str) -> list
+        """Get pools from Deadline.
+        Args:
+            webservice (str): Server url.
+            log (Logger)
+        Returns:
+            list: Pools.
+        Throws:
+            RuntimeError: If deadline webservice is unreachable.
+
+        """
+        if not log:
+            from openpype.lib import PypeLogger
+
+            log = PypeLogger().get_logger(__name__)
```

Comment:
Initialization of `PypeLogger` is not needed.
```suggestion
            log = PypeLogger.get_logger(__name__)
```